tests/python/pants_test/BUILD

```diff
@@ -58,6 +58,15 @@ python_library(
     'src/python/pants/util:process_handler',
     'tests/python/pants_test/testutils:file_test_util',
     'tests/python/pants_test/testutils:pexrc_util',
+    # NB: Integration tests in the pants codebase find the pants script by absolute path, which
```

Comment:
We don't currently chroot integration tests so mentioning chroot and absolute paths is a bit off point here. The main point is 'pants_run_integration_test.py' runs `./pants` in a subprocess, and so it properly depends on the pants binary and all of its transtitive dependencies. The 'src/python/pants/bin:pants_local_binary' dep is simply our best proxy for all that.

Reply:
That makes it much more clear, and I've changed the comment in 3ce1856.